### ISSUE TITLE
[BUG] 1.0.0: Build fails with fmt: no member named 'throw_format_error' in namespace 'fmt'

### ISSUE THREAD

**Issue by yurivict** [bluescarni/mppp#307]
**Describe the bug**
```
In file included from /usr/ports/math/mppp/work/mppp-1.0.0/src/detail/arb.cpp:46:
In file included from /usr/ports/math/mppp/work/mppp-1.0.0/include/mp++/real.hpp:51:
/usr/ports/math/mppp/work/mppp-1.0.0/include/mp++/detail/fmt.hpp:45:9: error: no member named 'throw_format_error' in namespace 'fmt'; did you mean 'fmt::detail::throw_format_error'?
        fmt::throw_format_error("Invalid format");
        ^~~~~~~~~~~~~~~~~~~~~~~
        fmt::detail::throw_format_error
/usr/local/include/fmt/core.h:630:27: note: 'fmt::detail::throw_format_error' declared here
FMT_NORETURN FMT_API void throw_format_error(const char* message);
                          ^
```

Version: 1.0.0
libfmt-9.1.0
clang-16
FreeBSD 13.2

**Comment by bluescarni** [comment]
Hi @yurivict thanks for the report.

Can you check if #308 fixes the issue?

**Comment by bluescarni** [comment]
This should now be fixed in version 1.0.1. Closing.
